rts/pom.xml

```diff
@@ -42,7 +42,7 @@
   </repositories>
   <modules>
     <module>at</module>
-    <module>lra</module>
+<!--    <module>lra</module>-->
```

Comment:
We need a comment that indicates why it's been disabled - a reference to the JBTM should be sufficient.

Reply:
okay done